src/listpack.c

```diff
@@ -540,9 +546,7 @@ lpGetWithSize(unsigned char *p, int64_t *count, unsigned char *intbuf, uint64_t
         if (entry_size) *entry_size = 5 + *count + lpEncodeBacklen(NULL, *count + 5);
         return p + 5;
     } else {
-        uval = 12345678900000000ULL + p[0];
-        negstart = UINT64_MAX;
-        negmax = 0;
+        panic("Invalid listpack encoding. Byte %02hhx is not a valid encoding.", p[0]);
```

Comment:
So I suppose we can't do this. While validating listpack entries, we need to fetch some of the data out to validate the integrity. However, if the listpack itself is itself malformed, we might try to extract the data out. The alternative would be to check every single point in `streamValidateListpackIntegrity` to verify we still have more data to read.